packages/office-ui-fabric-react/src/components/ComboBox/ComboBox.tsx

```diff
@@ -292,7 +293,7 @@ export class ComboBox extends BaseComponent<IComboBoxProps, IComboBoxState> {
           />
         </div>
 
-        { isOpen && (
+        { isOpen || (alwaysDisplaySuggestions && focused) && (
```

Comment:
consider putting the conditional in parenthesis to more clearly separate the pieces of this line of code